src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -39,6 +34,13 @@
 
 import lombok.extern.slf4j.Slf4j;
 
+
+
+import bisq.asset.Asset;
+import bisq.asset.AssetRegistry;
+import bisq.asset.Token;
+import bisq.asset.coins.BSQ;
+
```

Comment:
Please `--amend` this commit with fixed / re-organized imports.

I should have mentioned to you, @blabno, that I created bisq-network/bisq#1 with the correct configuration, and that you would have had to check out that pull request for everything to work correctly.

Because that's a bit painful, though—and because this is one of the "rough spots" around IDEA's support for composite builds—I've simply merged that PR eagerly now. So you can just pull the latest from `bisq-network/bisq:master` now, and you'll have the correct IDEA import layout table configuration. You should be able to simply re-organize imports and everything will "just work".

See https://github.com/bisq-network/style/issues/2 for further details and tips on this.